ldk/javascript/src/whisper/mapper.ts

```diff
@@ -4,6 +4,7 @@ import {
   Component,
   NewWhisper,
   StateMap,
+  Breadcrumbs,
```

Comment:
Correct me if I'm wrong, but I think we don't need to import Breadcrumbs, since we can use WhisperComponentType.Breadcrumbs below.  0.0

Reply:
When I tried that I got this error `Conversion of type 'Breadcrumbs' to type 'WhisperComponentType.Breadcrumbs' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first` and I was following what Box did on line 37

Reply:
I realized that you adjust Breadcrumbs's order a little bit, :) they are  alphabetical ordered